libs/ledgerjs/packages/crypto-utils/package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "@ledgerhq/crypto-utils",
```

Comment:
Is there a reason we make this a dedicated library, why not keeping this bip32 in hw-app-btc?  is it useful for another thing than bitcoin? looking at the code it is specific to bitcoin and xpub concepts.

If we still want to make this a dedicated library, i think we shouldn't recreate libs that have "common", "core" or "utils" in their name but directly tells what is the current scope of that lib.  at the moment the scope is a bit weird, what do you think?

Reply:
I am open to any feedback about the name of this package.

The purpose is to remove the hard dependency of "coin-bitcoin" to "hw-app-btc". For the time being I simply put the minimal code required to be shared, but we can add more.

I think we need to stop melting "blockhain specific logic", "hardware logic" and "cryptographic algorithm".
This last category is a tool for the others. But the first two have to be independent.